samples/java/tutorial_code/ShapeDescriptors/point_polygon_test/PointPolygonTestDemo.java

```diff
@@ -86,8 +84,6 @@ public void run() {
     public static void main(String[] args) {
         // Load the native OpenCV library
         System.loadLibrary(Core.NATIVE_LIBRARY_NAME);
-
         new PointPolygonTest().run();
     }
-
```

Comment:
Revert unnecessary changes (removal of empty lines).

> HighGui.imshow("Distance", drawing);

Need to update the name in Java sample too too.